cmd/run.go

```diff
@@ -95,13 +95,13 @@ var runCmd = &cobra.Command{
 		projectsDir := filepath.Join(home, HidePath, ProjectsDir)
 
 		lspServerExecutables := make(map[lsp.LanguageId]lsp.Command)
-		lspServerExecutables[lsp.LanguageId("go")] = lsp.NewCommand("gopls", []string{})
-		lspServerExecutables[lsp.LanguageId("python")] = lsp.NewCommand("pyright-langserver", []string{"--stdio"})
-		lspServerExecutables[lsp.LanguageId("javascript")] = lsp.NewCommand("typescript-language-server", []string{"--stdio"})
-		lspServerExecutables[lsp.LanguageId("typescript")] = lsp.NewCommand("typescript-language-server", []string{"--stdio"})
+		lspServerExecutables[lsp.Go] = lsp.NewCommand("gopls", []string{})
```

Comment:
I think we should move all this logic to `lsp` pkg. Something like `lsp.Excecutables() map[lsp.LanguageId]lsp.Command`

Reply:
done